website/src/styles/Theme/index.ts

```diff
@@ -0,0 +1,29 @@
+import { type ThemeConfig, extendTheme } from "@chakra-ui/react";
+import { containerTheme } from "./components/Container";
```

Comment:
Rather than import each components theme here can we create a `website/src/styles/Theme/components/index.ts` file.

With that we can simplify this as 
```
import * as components from "./components";
```

And then down below we don't need to define a `const components = { ...}`.  We can just use the import